system-tests/projects/runner-e2e-specs/cypress/e2e/errors/events.cy.js

```diff
@@ -1,18 +1,18 @@
 describe('event handlers', { defaultCommandTimeout: 0 }, () => {
   it('event assertion failure', () => {
-    cy.on('window:load', () => {
+    cy.on('command:end', () => {
```

Comment:
The docs don't specify this as CT only, we should update them at some point https://docs.cypress.io/api/events/catalog-of-events#App-Events

Reply:
The event itself works fine in CT, but the test was doing a `cy.visit` (which doesn't work in CT) and using this paired event listener. I had to replace the `cy.visit` with a different test condition, thus the change in events